Power_Recrsion.py

```diff
@@ -0,0 +1,44 @@
+def Power_Iterative(base, exponent):
+    result = 1
+    for i in range(exponent):
+        result *= base
+    print(result)
+
+
+# Power_Iterative(2,5)
+
+
+def Power_Recursive_n(base, exponent):
+    if exponent == 0:
+        return 1
+
+    return base * Power_Recursive_n(base, exponent - 1)
+
+
+# print(Power_Recursive_n(2,5))
+
+
+def Power_Recursive_logn(base, exponent):
```

Comment:
its too complex. As I mentioned the number of lines per function should not exceeed 10-12 lines.
We need to simplify it using wrapper function or decorator

Reply:
We need to do code in an efficient way at OCS, that's what we sell to our clients